content/800-data-platform/100-accelerate/580-local-development.mdx

```diff
@@ -0,0 +1,65 @@
+
+In development environments, you may want to use a local database to minimize expenses. Furthermore, you may consider extending `PrismaClient` with the Accelerate client extension once, so that you can use a local database in development and a hosted database with Accelerate’s connection pooling and caching enabled. This eliminates the need for conditional logic to switch clients between development and production.
+
+This guide will explain how to use Prisma Accelerate client extension in a development environment with a local database.
+
+</TopBlock>
+
+## Using Prisma Accelerate client extension in development and production
+
+<br />
+<img src="images/accelerate-in-dev.png" alt="" />
+
+Accelerate doesn’t work with a local database. However, using a local database URL, you can add the Accelerate client extension to your Prisma Client. This will allow you to use a local database in your development environment. Keep in mind that, in this scenario, caching and connection pooling won't be available.
+
+Here is how to use a local PostgreSQL database with Prisma ORM and Prisma Accelerate client extension (without caching and connection pooling):
+
+1. Paste your local database URL in your .env file
+
+   ```.env
+   DATABASE_URL="postgres://username:password@127.0.0.1:5432/localdb"
+   ```
+
+2. Generate a Prisma Client
+
+   ```bash
+   npx prisma generate
+   ```
+
+   > Note: The `--no-engine` cannot be used when working with a local database. The command generates Prisma Client artifacts without a [Query Engine](/concepts/components/prisma-engines) file, which requires the use of an Accelerate connection string.
+
+3. Add Prisma Accelerate to your project
+
+   ```typescript
+   import { PrismaClient } from '@prisma/client'
+   import { withAccelerate } from '@prisma/extension-accelerate'
+
+   const prisma = new PrismaClient().$extends(withAccelerate())
+   ```
+
+   > The extended instance of Prisma Client will use the local database. Hence, Prisma Accelerate will not be used in your development environment to respond to your Prisma Client queries.
+
+<img src="images/accelerate-in-prod.png" alt="" />
+
+In a production environment, when a generated Accelerate connection string is used, you will be able to use connection pooling and caching by adding the Accelerate client extension to the `PrismaClient`.
+
+## Locally using Prisma Accelerate in edge functions
+
+When using an edge runtime, e.g., [Next.js edge runtime](https://nextjs.org/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes#edge-runtime), for your development environment, update your Prisma Client import as follows:
+
+```typescript
+import { PrismaClient } from "@prisma/client/edge";
+```
+
+While edge function environments generally lack native support for existing APIs enabling TCP-based database connections, Prisma Accelerate provides a connection string that allows querying your database over HTTP, a protocol supported in all edge runtimes.
```

Comment:
```suggestion
Generally, edge function environments lack native support for existing APIs enabling TCP-based database connections.Prisma Accelerate provides a connection string that allows querying your database over HTTP, a protocol supported in all edge runtimes.
```

The smaller your sentences, the better. This would make it easier to read.